recipes/function2/all/conanfile.py

```diff
@@ -11,42 +16,53 @@ class Function2Conan(ConanFile):
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://github.com/Naios/function2"
     license = "BSL-1.0"
-    settings = "compiler"
+    settings = "os", "arch", "compiler", "build_type"
     no_copy_source = True
 
     @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
-    def configure(self):
-        minimal_cpp_standard = "14"
-        if self.settings.compiler.cppstd:
-            tools.check_min_cppstd(self, minimal_cpp_standard)
-        minimal_version = {
+    def _min_cppstd(self):
+        return 14
+
+    @property
+    def _compiler_minimal_version(self):
+        return {
             "gcc": "5",
             "clang": "3.4",
             "apple-clang": "10",
```

Comment:
There's a check min vs helper for the new msvc compiler see the package template https://github.com/conan-io/conan-center-index/blob/7e292bc025078633df6537fe50204423729c8672/docs/package_templates/cmake_package/all/conanfile.py#L90

Reply:
if it's a recurring pattern across many recipes, it would be beneficial to add helper/tool to conan client itself

Reply:
This is a 1.x c3i infra limitation for not supporting cppstd in the majority of cases

The hope is 2.0 we can remove most of these 🤞